config/initializers/s3.rb

```diff
@@ -1,11 +1,19 @@
 # frozen_string_literal: true
 
 CarrierWave.configure do |config|
-  config.fog_credentials = {
-    provider: 'AWS',
-    aws_access_key_id: ENV['AWS_ACCESS_KEY_ID'],
-    aws_secret_access_key: ENV['AWS_SECRET_KEY_ID']
-    # region: ENV['S3_REGION'] # Change this for different AWS region. Default is 'us-east-1'
-  }
-  config.fog_directory = ENV['S3_BUCKET']
+  if Rails.env.development?
+    config.storage = :file
+  else
+    config.fog_credentials = {
+      provider: 'AWS',
+      aws_access_key_id: ENV['AWS_ACCESS_KEY_ID'],
+      aws_secret_access_key: ENV['AWS_SECRET_KEY_ID']
+      # region: ENV['S3_REGION'] # Change this for different AWS region. Default is 'us-east-1'
+    }
+    config.storage = :fog
+    config.fog_directory = ENV['S3_BUCKET']
+
+    if Rails.env.test?
```

Comment:
@rlgreen91 ActiveStorage doesn't use `fog`, so I don't think it will work with `Fog::Mock`. It seems like we probably want to avoid stubbing the underlying implementation anyway, since that's not something we otherwise interface with directly in the app.